server/pbench/bin/index-pbench

```diff
@@ -2079,19 +2100,18 @@ class PbenchTarBall(object):
     def gen_run_timestamps(self):
         try:
             start_run_ts = datetime.strptime(self.start_run, "%Y-%m-%dT%H:%M:%S.%f")
-        except ValueError:
+        except ValueError as e:
```

Comment:
Should all the `except ValueError` be `except ValueError as e`? For consistency, if for nothing else.

Reply:
We won't use `e` any where, that was left over from a previous attempt to fix the problem that I abandoned.  I'll be refactoring this code in a bit and so this will be removed.